website/pages/en/developing/supported-networks.json

```
@@ -4,5 +4,6 @@
   "chainId": "Chain ID",
   "hostedService": "Hosted Service",
   "subgraphStudio": "Subgraph Studio",
-  "decentralizedNetwork": "Decentralized Network"
+  "decentralizedNetwork": "Decentralized Network",
+  "networkType": "Type"
```

Nitpick, but I don't see the point in this not just being
```suggestion
  "type": "Type"
```